base/scopedvalues.jl

```diff
@@ -102,6 +102,7 @@ function Scope(scope, pair1::Pair{<:ScopedValue}, pair2::Pair{<:ScopedValue}, pa
     # our compiler optimization support works
     return Scope(Scope(scope, pair1...), pair2, pairs...)
 end
+Scope(scope::Scope) = scope
```

Comment:
I just PR'd this fix specifically https://github.com/JuliaLang/julia/pull/56019

Reply:
Let's merge that ASAP, as it should get backported